src/test/java/seedu/address/logic/commands/FeedCommandTest.java

```diff
@@ -81,15 +83,6 @@ public void execute_remoteUrl_success() throws Exception {
         assertFeedSuccessfullyLoaded(TEST_URL);
     }
 
-    @Test
-    public void execute_malformedUrlGiven_commandFails() {
```

Comment:
No more malformed URL test since it is now impossible to even construct the url argument to give to the FeedCommand (in other words, it will fail earlier, during parsing).